styles/main.css

```
@@ -1,14 +1,475 @@
+      height: 2px;
+      border-radius: 2px; }
+    @media (max-width: 950px) {
+      .promo__info {
+        margin-left: 10px; } }
+  .promo__title {
+    font-size: 60px; }
+
+@media (max-width: 950px) {
+  .promo__link {
+    display: none; }
+  .promo__call {
+    display: block;
+    opacity: 100; } }
+
+.nav {
+  text-transform: uppercase;
+  margin: 0 120px; }
+  .nav__list {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+        -ms-flex-pack: justify;
+            justify-content: space-between; }
+    @media (max-width: 950px) {
+      .nav__list {
+        position: absolute;
+        -webkit-box-orient: vertical;
+        -webkit-box-direction: normal;
+            -ms-flex-direction: column;
+                flex-direction: column;
+        -ms-flex-pack: distribute;
+            justify-content: space-around;
+        height: 100%;
+        background-color: blue;
+        padding-top: 120px;
+        top: 0;
+        right: 0;
+        display: none; } }
+  .nav__toggler {
+    display: none; }
+  .nav__toggler:checked ~ .nav__list {
+    display: block; }
+  .nav__toggler-label {
+    display: none;
+    position: absolute;
+    top: 42px;
+    right: 120px;
+    z-index: 2; }
+    .nav__toggler-label::after {
+      content: "\2630";
+      height: 12px;
+      width: 18px;
+      color: white;
+      font-size: 24px; }
+    @media (max-width: 950px) {
+      .nav__toggler-label {
+        display: block; } }
+  .nav__toggler:checked + .nav__toggler-label::after {
+    content: "x"; }
+  .nav__item {
+    padding: 8px 24px;
+    border-radius: 8px;
+    list-style: none;
+    -webkit-transition: 0.3s;
+    transition: 0.3s; }
+    .nav__item-active {
+      background-color: rgba(255, 255, 255, 0.2); }
+    .nav__item:hover,
+    .nav__item :active {
+      opacity: 0.8; }
+    @media (max-width: 950px) {
+      .nav__item {
+        margin-bottom: 70px; } }
+  .nav__link {
+    color: #fff;
+    text-decoration: none;
+    letter-spacing: 2px; }
+
+.section {
+  margin: 0 120px 152px; }
+  @media (max-width: 950px) {
+    .section {
+      margin: 10px; } }
+  .section__title {
+    display: block;
+    margin-bottom: 93px;
+    text-align: center;
+    font-size: 48px;
+    line-height: 48px;
+    color: #000; }
+    .section__title::before {
+      content: "";
+      width: 23px;
+      height: 16px;
+      position: relative;
+      top: 70px;
+      left: 50%;
+      -webkit-transform: translateX(-50%);
+              transform: translateX(-50%);
+      display: block;
+      background: url(../images/crown.svg) center no-repeat; }
+
+.info {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between; }
+  .info__title {
+    color: black;
+    font-size: 48px;
+    letter-spacing: -1.5px; }
+    .info__title-active {
+      color: blue;
```

I guess, in the mockup, we have another color
```suggestion
      color: #0075ff;
```